Assets/Scripts/Utilities/InputUtils.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public static class InputUtils
+{
+	// TODO ask Chris what this is even for
+	public static void CheckForController()
```

Comment:
If you put this in an update loop somewhere it will change the platformName to match whether a controller is plugged in or not once per second.